metricbeat/module/mysql/mysql.go

```diff
@@ -1,34 +1,19 @@
+/*
+Package mysql is Metricbeat module for MySQL server.
+*/
 package mysql
 
 import (
 	"database/sql"
 
-	"github.com/elastic/beats/metricbeat/helper"
-
+	// Register the MySQL driver.
 	_ "github.com/go-sql-driver/mysql"
 )
 
-func init() {
```

Comment:
Yeah, this remove the boilerplate code from the Modules when it is not needed. I added an example of how to register a custom ModuleFactory here. https://godoc.org/github.com/andrewkroh/beats/metricbeat/mb#ModuleFactory